docs/docusaurus.config.js

```diff
@@ -81,13 +81,21 @@ const config = {
             title: 'Docs',
             items: [
               {
-                label: 'User Guide',
+                label: 'Draft User Guide',
```

Comment:
Isn't everything a draft at this point?

Reply:
Yes; I think it probably benefits from being labeled as such as much as possible.